src/modules/components/proposal/proposalActions/proposalActionsList/proposalActionChangeMembers/proposalActionChangeMembers.tsx

```diff
@@ -1,13 +1,19 @@
 import { DefinitionList, Heading } from '../../../../../../core';
+import { ChainEntityType, useBlockExplorer } from '../../../../../hooks';
```

Comment:
I'm seeing many warnings on tests introduced by this PR, can you check why?
`Error: Not implemented: HTMLCanvasElement.prototype.getContext (without installing the canvas npm package)`